client/api/python/test/unit/heketi.json

```diff
@@ -34,6 +34,8 @@
       "user": "sshuser"
     },
 
+    "loglevel" : "debug",
```

Comment:
This is something which I am not sure about:
1. it is good to have debug logs ready when a test fails
2. it might however mask some race conditions as debug logs add enough delay to operations.

I am a little inclined towards default log level in tests too. Let developers run in debug level the second time when a test fails.

@heketi/dev what do others think?